bor/keeper.go

```diff
@@ -45,7 +45,7 @@ type Keeper struct {
 	chainKeeper chainmanager.Keeper
 }
 
-// NewKeeper create new keeper
+// NewKeeper is the constructor of Keeper
```

Comment:
Could we remove such comments, how do you think? I don't see much afford in commenting obvious things.

Reply:
according to the general codestyle, exported types should have comments, even if it is obvious

Reply:
do we have 'general codestyle' somewhere?

Reply:
I believe we don't have it yet, but would be nice to have it. This is a kind of good practice to comment the code. Sometimes it might seem redundant because the code is obvious. Just following the common practice to comment exported types.